durst/src/main.rs

```diff
@@ -67,20 +70,23 @@ impl interface::OrgFreedesktopNotifications for Mutex<Notifications> {
     }
 }
 
-impl AsRef<dyn interface::OrgFreedesktopNotifications + 'static> for Rc<Mutex<Notifications>> {
+impl AsRef<dyn interface::OrgFreedesktopNotifications + 'static> for Rc<Mutex<Container>> {
     fn as_ref(&self) -> &(dyn interface::OrgFreedesktopNotifications + 'static) {
         &**self
     }
 }
 
 fn run() -> Result<(), Box<dyn std::error::Error>> {
-    let notifications_rc = Rc::new(Mutex::new(Notifications {
+    let tmp = config::load_config("~/.config/durst/config.yaml".to_string());
```

Comment:
The important thing: This should be the `$XDG_CONFIG_DIR/durst/config.yaml`.

It's just that XDG_CONFIG_DIR is unset on most systems and defaults to `~/.config` if unset.

There might be a proper XDG lib in Rust. This might seem to the be the right method: https://docs.rs/dirs/1.0.4/dirs/fn.config_dir.html

Reply:
Just as a personal preference, I'd go for ".yml" instead of ".yaml".

Reply:
have a look at the new commit. I do not think we need this dependecy